spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -221,6 +221,8 @@ class CodeEditor(TextEditBaseWidget):
 
     sig_breakpoints_changed = Signal()
     sig_debug_stop = Signal(int)
+    sig_stop_debugging = Signal()

```

Comment:
Since we already have a signal called `sig_debug_stop`, let's use that signal instead of creating a new one with almost the same name (`sig_stop_debugging`).

The thing is you can have a signal with two different signatures and connect different slots to each signature. For that you need to define `sig_debug_stop` like this:

```python
sig_debug_stop = Signal((int,), ())
```

The first signature will be the default one, so you don't need to declare it when emitting from this signal. In other words, you can simply do

```python
sig_debug.stop.emit(1)
```

But you need to declare the signature when you want to emit from the second one, like this

```python
sig_debug.stop[()].emit()
```

I'll leave a comment in `editor/plugin.py` to show you how that's done in this case.